.github/workflows/test-template.yml

```
@@ -990,7 +990,7 @@ jobs:
   # Driver Adapter Unit Tests
   #
   driver-adapter-unit-tests:
-    name: Driver Adapter Unit Tests
+    name: DA Unit Tests [v${{ matrix.node }}]
```

```suggestion
    name: Driver Adapter [v${{ matrix.node }}]
```
I think it's more useful to highlight that it's a Driver Adapter job than that its a unit test

---

@FGoessler I don't like the DA acronym, I don't think we use it anywhere so it's not obvious what it means here

---

Any other ideas how we can make the tests readable in those sidebars? 🤔 
If I just read "Tests / Driver Adapters..." repeated 20+ times I have no value of it and cannot navigate test failures efficiently :/